### ISSUE TITLE
Implement in-memory caching

### ISSUE THREAD

**Issue by bobateadev** [PaddlePaddle/PaddlePaddle.org#4]

**Comment by varunarora** [comment]
What do we still want cached? Sitemaps?

**Comment by cs2be** [comment]
Yes, sitemaps will still be cached (because we still have to do some post processing).  We may also want to cache URLs on AWS side later, we can put that as another task.

**Comment by varunarora** [comment]
Okay, so I can setup AWS Elastic Cache for this and give you all the info to get you started (with perhaps an example). Would that be helpful?
